docker-compose.yaml

```diff
@@ -45,6 +45,8 @@ services:
       DB_NAME: ${DB_NAME}
       DB_SCHEMA: /app_sqlschema/schema.sql
       SEED_MODE: ${SEED_MODE}
+      AUTH_TYPE: token
+      AUTH_TOKEN_MAP: '{"testscanner":"xxx", "someotherscanner":"yyy"}'
```

Comment:
As discussed lets use 1 Secret for all scanners.

Reply:
Done